jax/experimental/jax2tf/saved_model_lib.py

```diff
@@ -21,47 +21,32 @@
 
 from typing import Callable, Sequence, Optional
 
-from jax.experimental import jax2tf # type: ignore[import]
+from jax.experimental import jax2tf  # type: ignore[import]
 import tensorflow as tf  # type: ignore[import]
 
 
-def save_model(jax_fn: Callable,
-               params,
-               model_dir: str,
-               *,
-               input_signatures: Sequence[tf.TensorSpec],
-               shape_polymorphic_input_spec: Optional[str] = None,
-               with_gradient: bool = False,
-               enable_xla: bool = True,
-               compile_model: bool = True):
-  """Saves the SavedModel for a function.
+def convert_and_save_model(jax_fn: Callable,
+                           params,
+                           model_dir: str,
+                           *,
+                           input_signatures: Sequence[tf.TensorSpec],
+                           shape_polymorphic_input_spec: Optional[str] = None,
+                           with_gradient: bool = False,
+                           enable_xla: bool = True,
+                           compile_model: bool = True,
+                           save_model_options=None):
```

Comment:
We can hint this: `: tf.saved_model.SaveOptions`

Reply:
Done